docs/scos/dev/feature-integration-guides/202204.0/glue-api/glue-backend-api/glue-api-authorization-protected-endpoints-integration.md

```diff
@@ -0,0 +1,204 @@
+
+```bash
+composer require spryker/glue-storefront-api-application-authorization-connector:"^1.0.0" \
+spryker/glue-backend-api-application-authorization-connector:"^1.0.0" \
+--update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following modules have been installed:
+
+| MODULE | EXPECTED DIRECTORY |
+| --- | --- |
+| GlueStorefrontApiApplicationAuthorizationConnector | vendor/spryker/glue-storefront-api-application-authorization-connector|
+| GlueBackendApiApplicationAuthorizationConnector | vendor/spryker/glue-storefront-api-application-authorization-connector|
+
+{% endinfo_block %}
+
+### 2) Set up transfer objects
+
+Generate transfers:
+
+```bash
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied in transfer objects:
+
+| TRANSFER | TYPE | EVENT | PATH |
+| --- | --- | --- | --- |
+| AuthorizationRequest | class | created | src/Generated/Shared/Transfer/AuthorizationRequestTransfer.php |
+| AuthorizationResponse | class | created | src/Generated/Shared/Transfer/AuthorizationResponseTransfer.php |
+| RouteAuthorizationConfig | class | created | src/Generated/Shared/Transfer/RouteAuthorizationConfigTransfer.php |
+| GlueResourceMethodConfiguration | class | created | src/Generated/Shared/Transfer/GlueResourceMethodConfigurationTransfer.php |
+| GlueResourceMethodCollection | class | created | src/Generated/Shared/Transfer/GlueResourceMethodCollectionTransfer.php |
+| ResourceContext | class | created | src/Generated/Shared/Transfer/ResourceContextTransfer.php |
+| CustomRoutesContext | class | created | src/Generated/Shared/Transfer/CustomRoutesContextTransfer.php |
+| PathMethodComponentData | class | created | src/Generated/Shared/Transfer/PathMethodComponentDataTransfer.php |
+| AuthorizationEntity | class | created | src/Generated/Shared/Transfer/AuthorizationEntityTransfer.php |
+| GlueRequest | class | created | src/Generated/Shared/Transfer/GlueRequestTransfer.php |
+| Route | class | created | src/Generated/Shared/Transfer/RouteTransfer.php |
+| ApiApplicationSchemaContext | class | created | src/Generated/Shared/Transfer/ApiApplicationSchemaContextTransfer.php |
+
+{% endinfo_block %}
+
+### 2) Set up behavior
+
+Activate the following plugins:
+
+**Storefront API plugins**
+
+| PLUGIN | SPECIFICATION                                                                        | NAMESPACE                                                                                            |
+| --- |--------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| IsProtectedTableColumnExpanderPlugin | Provides capability to extends route description table.                              | Spryker\\Glue\\GlueStorefrontApiApplicationAuthorizationConnector\\Plugin\\GlueApplication           |
```

Comment:
It's a description that is valid for the plugin interface, not the plugin. Please adjust to a more specific one. (Applicable to BackendApi as well)